tests/test_enterprise/api/test_views.py

```diff
@@ -2283,6 +2286,77 @@ def test_post_request_codes_permissions(self, implicit_perm, explicit_perm, expe
 
         assert response.status_code == expected_status
 
+    @mock.patch('enterprise.api.v1.views.EnrollmentApiClient')
+    @mock.patch('enterprise.api.v1.views.get_certificate_for_user')
+    @mock.patch('enterprise.api.v1.views.get_course_overviews')
+    @ddt.data(
+        (True, False, True, status.HTTP_204_NO_CONTENT),
```

Comment:
Nit: `ddt.unpack` will unpack a dict if you want to write these as
```python
{'has_permissions': True, 'is_courrse_enrollment_completed': False, 'is_licensed_enrollment_revoked': True, 'expected_status_code': status.HTTP_204_NO_CONTENT}
```
...which I personally find helpful, but is maybe a matter of taste.